docs/examples/static-website/tutorial.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Static Websites
+---
+
+## Quick-start tutorial
+
+This tutorial walks you through the process of deploying a _static website_ on KintoHub.
+
+There are two examples, one with standard HTML files, and one where the static website is generated with a framework, in this case [Angular](https://angular.io).
+
+## Static HTML website
+
+<video autoplay loop muted controls width="100%">
+   <source src="/docs/assets/examples/static-websites/html.mp4"
+           type="video/mp4">
+   Sorry, your browser doesn't support embedded videos.
+</video>
+
+### Create a Website KintoBlock
```

Comment:
Is it intentional to have two "Create a website kintoblock" sections? Or is that extra.